bioptim/interfaces/acados_interface.py

```diff
@@ -166,6 +166,9 @@ def __set_costs(self, ocp):
         self.mayer_costs = SX()
         self.W = np.zeros((0, 0))
         self.W_e = np.zeros((0, 0))
+        exclude_lagrange_to_convert_in_mayer = ['MINIMIZE_TORQUE', 'TRACK_TORQUE', 'MINIMIZE_TORQUE_DERIVATIVE',
+                                                'MINIMIZE_MUSCLES_CONTROL', 'TRACK_MUSCLES_CONTROL',
+                                                'MINIMIZE_ALL_CONTROLS', 'TRACK_ALL_CONTROLS']
```

Comment:
Is that necessary? It seems that all these do not define the last node anyway since they are controls? Would not this be more general to test if the last node is actually used?
I assume there is later a place where the values are dispatched, it would be there (by comparing the dimensions of the objective function)